### PR TITLE
Updates - Read Below

### LINKED ISSUE #3733: Add: Pennsylvania Cable Network
PCN Pennsylvania Cable Net

**Channel Name:** PCN Pennsylvania Cable Net 
**Country:** US
**Language:** English
**Website:** bozztv.com
**Stream URL (optional):** https://bozztv.com/teleyupp/teleup-pcnhd/tracks-v2a1/mono.m3u8
**Notes (optional):** xxx

### PR DESCRIPTION
**Pull request by MapGuy11**
1. Change MeTV West (I assume) to MeTV Arizona from (Distributed Antenna System)
2. Add Pennsylvania Cable Network from (BozzTV/TeleupTV)
3. Add Entertainment Studios Channels from (BozzTV/TeleupTV)
4. Add Popular Digital Subchannels from (Distributed Antenna System)
5. Add 2 **NEW** Subchannels in the US _True Real_ and _Defy TV_

Closes https://github.com/iptv-org/iptv/issues/3733

### CODE REVIEW

File: channels/us.m3u [hunk 1]
```diff
@@ -6558,9 +6558,39 @@ http://188.40.68.167/russia/disney/playlist.m3u8
...
+#EXTINF:-1 tvg-id="KAZTDT2.us" tvg-name="MeTV Arizona" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/6nXpZqz.png" group-title="News",MeTV Arizona
 https://streams.the6tv.duckdns.org:2443/locals/Pheonix/kazt-dt2.m3u8
 #EXTINF:-1 tvg-id="KAZTDT3.us" tvg-name="KAZT-DT3" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/6nXpZqz.png" group-title="News",KAZT-DT3 (HSN)
 https://streams.the6tv.duckdns.org:2443/locals/Pheonix/kazt-dt3.m3u8
 #EXTINF:-1 tvg-id="KAZTDT5.us" tvg-name="KAZT-DT5" tvg-country="US" tvg-language="English" tvg-logo="" group-title="News",KAZT-DT5 (AZTV 7 - Encore)
-https://streams.the6tv.duckdns.org:2443/locals/Pheonix/kazt-dt5.m3u8
\ No newline at end of file
+https://streams.the6tv.duckdns.org:2443/locals/Pheonix/kazt-dt5.m3u8
+#EXTINF:-1 tvg-id="PCN.us" tvg-name="Pennsylvania Cable Network" tvg-country="US" tvg-language="English" tvg-logo="https://zap2it.tmsimg.com/h3/NowShowing/11044/s11044_h3_aa.png" group-title="News",Pennsylvania Cable Network
+https://bozztv.com/teleyupp/teleup-pcnhd/playlist.m3u8
```
Comment (Dum4G) [inline channels/us.m3u:6568]: Every BozzTV link is Geo-Blocked

File: channels/us.m3u [hunk 2]
```diff
@@ -6558,9 +6558,39 @@ http://188.40.68.167/russia/disney/playlist.m3u8
...
+#EXTINF:-1 tvg-id="CARSTV.us" tvg-name="Cars.TV" tvg-country="US" tvg-language="English" tvg-logo="https://zap2it.tmsimg.com/h3/NowShowing/71302/s71302_h3_aa.png" group-title="Entertainment",Cars.TV
```
Comment (Dum4G) [inline channels/us.m3u:6569]: Sounds like Auto category

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/us.m3u (hunk 1 above)
Comment (MapGuy11) [reply channels/us.m3u:6568]: Will Fix

File: channels/us.m3u (hunk 2 above)
Comment (MapGuy11) [reply channels/us.m3u:6569]: Will Fix

**Comment by Dum4G** [comment]
I'll check every BozzTV channel for geo-block and resolve merge conflict first. Channels you have added didn't open for me but KTLA5 works just fine

**Comment by Dum4G** [comment]
@LaneSh4d0w  I'v got an interesting behavior to check. I was able to watch every BozzTV channel with teleyupp in links just fine, but when I tried one with teleyupp1 it started to restrict me. Please check if you can reproduce the same

**Comment by LaneSh4d0w** [comment]
> @LaneSh4d0w  I'v got an interesting behavior to check. I was able to watch every BozzTV channel with teleyupp in links just fine, but when I tried one with teleyupp1 it started to restrict me everywhere. Please check if you can reproduce the same
> 
> UPD: as expected, it was a temporal block

I'll check that, had a few things to do.

**Comment by LaneSh4d0w** [comment]
Links I've checked worked fine, no blockage whatsoever @Dum4G

**Comment by Dum4G** [comment]
> Links I've checked worked fine, no blockage whatsoever @Dum4G

Even if you try teleyupp1 link and then smth else?

**Comment by LaneSh4d0w** [comment]
Yes, actually i've alternated between teleyupp and teleyupp1, then going back to teleyupp, and all of them worked fine (tested C-Span, Fox 7 and Freeform)

**Comment by Dum4G** [comment]
I'm removing geo-blocked tag then

**Comment by Dum4G** [comment]
At this point I don't want to expirement with Github Desktop merge implementation. I'll do all of the changes from here in a new PR

**Pull request closed without merging**